modules/alloy/src/main/ts/ephox/alloy/positioning/layout/LayoutLabels.ts

```diff
@@ -0,0 +1,44 @@
+const southEast = 'tox-layout-se';
+const southWest = 'tox-layout-sw';
+const northEast = 'tox-layout-ne';
+const northWest = 'tox-layout-nw';
+const north = 'tox-layout-n';
+const south = 'tox-layout-s';
+const east = 'tox-layout-e';
+const west = 'tox-layout-w';
+
+const southEastInner = 'tox-layout-inner-se';
+const southWestInner = 'tox-layout-inner-sw';
+const northEastInner = 'tox-layout-inner-ne';
+const northWestInner = 'tox-layout-inner-nw';
+const northInner = 'tox-layout-inner-n';
+const southInner = 'tox-layout-inner-s';
+const eastInner = 'tox-layout-inner-e';
+const westInner = 'tox-layout-inner-w';
+
+const northPinned = 'tox-layout-pinned-n';
+const southPinned = 'tox-layout-pinned-s';
+
+const contextBarTransitionClass = 'tox-context-bar-layout-transition-animation';
```

Comment:
This 100% cannot be here either, as this is specific to TinyMCE/Silver. It needs to be there, not in Alloy.

Reply:
What'd be the correct way to do this? Simply name it something else?

Reply:
You need to make it so the class is passed in, as Alloy needs to be configurable and can be used anywhere. I haven't looked at this enough and am still away so I won't be able to look into it and give suggestion until next week sorry.

Reply:
We ended up doing just that.